src/SelfAssessment/Guidance.tsx

```diff
@@ -183,8 +183,6 @@ const Guidance: FunctionComponent<GuidanceProps> = ({
     }
   }
 
-  const displayFindATestCenter = Boolean(findATestCenterUrl)
```

Comment:
Why was this removed? what happens if the jurisdiction does not have a test center url

Reply:
good catch 👍